codec-http/src/main/java/io/netty/handler/codec/http/HttpContentEncoder.java

```diff
@@ -57,6 +57,7 @@
     }
 
     private final Queue<CharSequence> acceptEncodingQueue = new ArrayDeque<CharSequence>();
+    private final Queue<HttpMethod> httpMethodQueue = new ArrayDeque<HttpMethod>();
```

Comment:
I wonder if we could just store both the method and the encoding into the same queue. WDYT ?